app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -25,26 +32,58 @@
 public class MainActivity extends AppCompatActivity {
 
     private int levelsCount;
-    private int currentLevel;
+    private SharedPreferences gameData;
+    private SharedPreferences.Editor gameDataEditor;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        gameData = getSharedPreferences(GAME_VIEW_DATA, Context.MODE_PRIVATE);
+        gameDataEditor = gameData.edit();
         levelsCount = GameConstants.GAME_LEVELS_COUNT;
-        currentLevel = 1;
         setContentView(R.layout.main_activity);
-
     }
 
     public int getLevelsCount() {
         return levelsCount;
     }
 
-    public int getCurrentLevel() {
-        return currentLevel;
+    public void increaseLevel() {
```

Comment:
It is hard to understand what is this method doing from its name. What is "increase" level and what level actually gets increased?

From what I understand, this method is for the case when player has completed the level, and we need to record that their current level is the next one. Could we call it "onCurrentLevelCompleted" instead? Or something like that.

Reply:
Renamed